File: modules/Volo/modules/Volo.Chat/src/Volo.Chat.Application.Contracts/Volo/Chat/Messages/SendMessageInput.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Volo.Chat.Messages
{
    public class SendMessageInput
    {
        public Guid TargetUserId { get; set; }

        [Required]
        [StringLength(ChatMessageConsts.MaxTextLength, MinimumLength = 1)]
        public string Message { get; set; }
    }
}

File: modules/Volo/modules/Volo.Chat/angular/projects/chat/src/lib/models/send-message-input.ts

export class SendMessageInput  {
  targetUserId: string;
  message: string;

  constructor(initialValues: Partial<SendMessageInput> = {}) {
    if (initialValues) {
      for (const key in initialValues) {
        if (initialValues.hasOwnProperty(key)) {
          this[key] = initialValues[key];
        }
      }
    }
  }
}
